src/client/java/teammates/client/scripts/scalabilitytests/InstructorCourseEnrollPageScaleTestDataGenerator.java

```diff
@@ -0,0 +1,50 @@
+package teammates.client.scripts.scalabilitytests;
+
+import org.kohsuke.randname.RandomNameGenerator;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.HashSet;
+
+/**
+ * Generates test data for InstructorCourseEnrollPageScaleTest.
+ */
+public final class InstructorCourseEnrollPageScaleTestDataGenerator {
+
+    private static final String FILENAME = "InstructorCourseEnrollPageScaleTestData";
+
+    private static final String HEADER = "Section | Team | Name | Email | Comments\n";
+
+    private static final int[] studentGroups = {10, 20, 50, 75, 100, 150};
+
+    private InstructorCourseEnrollPageScaleTestDataGenerator() {
+    }
+
+    @SuppressWarnings("PMD.ConsecutiveLiteralAppends")
+    private static String generateStudents(int num) {
+        RandomNameGenerator generator = new RandomNameGenerator(num);
+        HashSet<String> studentNames = new HashSet<>();
+        StringBuffer students = new StringBuffer(HEADER);
+        for (int i = 0; i < num; i++) {
+            String curName = generator.next();
+            while (studentNames.contains(curName)) {
+                curName = generator.next();
+            }
+            String curUsername = curName.toLowerCase().replace("_", ".");
+            students.append("Section " + (i / 100 + 1) + " | Team " + (i / 100 + 1) + " | " + curName + " | "
```

Comment:
Maybe you could use `String.format` to look neat and avoid consecutive literal appends.